arxlive-search/src/App.js

```diff
@@ -1,35 +1,64 @@
 import React, { Component } from 'react'
+
 import extend from 'lodash/extend'
 import { SearchkitManager,SearchkitProvider,
-  SearchBox, RefinementListFilter, Pagination,
-  HierarchicalMenuFilter, HitsStats, SortingSelector, NoHits,
-	 ResetFilters, RangeFilter, NumericRefinementListFilter, ViewSwitcherHits, ViewSwitcherToggle, DynamicRangeFilter, Hits, HitItemProps,
-  InputFilter, GroupedSelectedFilters,
-  Layout, TopBar, LayoutBody, LayoutResults,
-  ActionBar, ActionBarRow, SideBar } from 'searchkit'
+	 SearchBox, RefinementListFilter, Pagination,
+	 HierarchicalMenuFilter, HitsStats, SortingSelector, NoHits,
+	 CheckboxFilter, ResetFilters, RangeFilter,
+	 NumericRefinementListFilter, ViewSwitcherHits, ViewSwitcherToggle,
+	 DynamicRangeFilter, Hits, HitItemProps, BoolMust,
+	 RangeQuery, HierarchicalRefinementFilter,
+	 InputFilter, GroupedSelectedFilters,
+	 Layout, TopBar, LayoutBody, LayoutResults,
+	 ActionBar, ActionBarRow, SideBar } from 'searchkit'
+
+import { DateRangeFilter, DateRangeCalendar } from "searchkit-datefilter"
+import * as moment from "moment";
+
 import './index.css'
+import '../node_modules/searchkit-datefilter/release/theme.css'
+
+import 'katex/dist/katex.min.css'
+import Latex from 'react-latex-next'
 
 
-const host = "https://lf3922vot1.execute-api.eu-west-2.amazonaws.com/v00/cliosearch/meetup_v2/"
+const host = "https://lf3922vot1.execute-api.eu-west-2.amazonaws.com/v00/cliosearch/arxiv_v0/"
+
 const searchkit = new SearchkitManager(host, {
     httpHeaders:{"Content-Type":"application/json",
-		 "X-Api-Key":"wXLnActJhY7r0XUDIWNXc6y6tGYUUnBU1eeU7Stu",
-		 "Es-Endpoint":"search-health-scanner-5cs7g52446h7qscocqmiky5dn4.eu-west-2.es.amazonaws.com",
-		}
+                 "X-Api-Key":"wXLnActJhY7r0XUDIWNXc6y6tGYUUnBU1eeU7Stu",
```

Comment:
does this stuff need to be kept private?

Reply:
and the endpoint?

Reply:
Good point - added issue #10 . For MVP not completely necessary, since these will be publicly viewable at runtime.